kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -341,6 +366,10 @@
       },
     },
     $trs: {
+      libraryOf: {
```

Comment:
@marcellamaki @radinamatic flagging the string movement here and in the next file.